docs/getting_started/Playground.md

```diff
@@ -54,13 +56,18 @@ The goal is to give you a taste on the most interesting and impactful aspects of
 
     </details>
 
+## Setup Workspace
+   ```
+	export WORKDIR=$HOME/work/graph-test
```

Comment:
@sankar17 Can we just not sure the PWD to create the directory ?
Let the user decide where the command needs to be rung ?
Asking for WORKDIR is still one more detail which is better avoided ?

Reply:
it might be better to explicitly call it out, pwd keeps changing so more ways that the user can make mistake. my 2 cents.

Reply:
Sure, sounds good.